tracer/README.MD

```diff
@@ -1,20 +1,115 @@
+    - .NET desktop development
+    - .NET Core cross-platform development
+    - Optional: ASP.NET and web development (to build samples)
+  - Individual components
+    - .NET Framework 4.7 targeting pack
+- [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+- [.NET 6.0 x86 SDK](https://dotnet.microsoft.com/download/dotnet/6.0) to run 32-bit tests locally
+- Optional: [ASP.NET Core 2.1 Runtime](https://dotnet.microsoft.com/download/dotnet-core/2.1) to test in .NET Core 2.1 locally.
+- Optional: [ASP.NET Core 3.0 Runtime](https://dotnet.microsoft.com/download/dotnet-core/3.0) to test in .NET Core 3.0 locally.
+- Optional: [ASP.NET Core 3.1 Runtime](https://dotnet.microsoft.com/download/dotnet-core/3.1) to test in .NET Core 3.1 locally.
+- Optional: [nuget.exe CLI](https://www.nuget.org/downloads) v5.3 or newer
+- Optional: [WiX Toolset 3.11.1](http://wixtoolset.org/releases/) or newer to build Windows installer (msi)
+  - [WiX Toolset Visual Studio Extension](https://wixtoolset.org/releases/) to build installer from Visual Studio
+- Optional: [Docker for Windows](https://docs.docker.com/docker-for-windows/) to build Linux binaries and run integration tests on Linux containers. See [section on Docker Compose](#building-and-running-tests-with-docker-compose).
+  - Requires Windows 10 (1607 Anniversary Update, Build 14393 or newer)
+
+Microsoft provides [evaluation developer VMs](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines) with Windows 10 and Visual Studio pre-installed.
+
+#### Building from a command line
+
+This repository uses [Nuke](https://nuke.build/) for build automation. To see a list of possible targets run:
+
+```cmd
+.\build.cmd --help
+```
+
+For example:
+
+```powershell
+# Clean and build the main tracer project
+.\build.cmd Clean BuildTracerHome
+
+# Build and run managed and native unit tests. Requires BuildTracerHome to have previously been run
+.\build.cmd BuildAndRunManagedUnitTests BuildAndRunNativeUnitTests 
+
+# Build NuGet packages and MSIs. Requires BuildTracerHome to have previously been run
+.\build.cmd PackageTracerHome 
+
+# Build and run integration tests. Requires BuildTracerHome to have previously been run
+.\build.cmd BuildAndRunWindowsIntegrationTests
+```
+
+### Linux
+
+The recommended approach for Linux is to build using Docker. You can use this approach for both Windows and Linux hosts. The _build_in_docker.sh_ script automates building a Docker image with the required dependencies, and running the specified Nuke targets. For example:
+
+```bash
+# Clean and build the main tracer project
+./build_in_docker.sh Clean BuildTracerHome
+
+# Build and run managed unit tests. Requires BuildTracerHome to have previously been run
+./build_in_docker.sh BuildAndRunManagedUnitTests 
+
+# Build and run integration tests. Requires BuildTracerHome to have previously been run
+./build_in_docker.sh BuildAndRunLinuxIntegrationTests
+```
+
+### MacOs
```

Comment:
We don't officially support macOS. Do we want this section?

Reply:
It's good to include for building, maybe it just needs a disclaimer that it is not officially supported. And you could even link to the main docs page that shows all supported frameworks/runtimes/OSes